arithmetization/src/test/java/net/consensys/linea/zktracer/module/hub/TxSkipTests.java

```diff
@@ -200,4 +199,120 @@ void test() {
         .build()
         .run();
   }
+
+  @Test
+  void receiverIsCoinbase() {
+
+    final KeyPair senderKeyPair = new SECP256K1().generateKeyPair();
+    final Address senderAddress =
+        Address.extract(Hash.hash(senderKeyPair.getPublicKey().getEncodedBytes()));
+    final ToyAccount senderAccount =
+        ToyAccount.builder().balance(Wei.fromEth(123)).nonce(5).address(senderAddress).build();
+
+    final ToyAccount coinbaseAccount =
+        ToyAccount.builder()
+            .address(DEFAULT_COINBASE_ADDRESS)
+            .balance(Wei.fromEth(2))
+            .nonce(5)
+            .build();
+
+    final Transaction tx =
+        ToyTransaction.builder()
+            .sender(senderAccount)
+            .to(coinbaseAccount)
+            .keyPair(senderKeyPair)
+            .value(Wei.of(123))
+            .build();
+
+    ToyExecutionEnvironmentV2.builder()
+        .accounts(List.of(coinbaseAccount, senderAccount))
+        .transaction(tx)
+        .zkTracerValidator(zkTracer -> {})
+        .build()
+        .run();
+  }
```

Comment:
Does this work for you ? We had a regression in Besu with @ahamlat / @Gabriel-Trintinalia / @lu-pinto's fix being undone in a recent commit, which may make this test fail now.